### PR TITLE
Add edit title/description screen

### PR DESCRIPTION
**Pull request by AndreVero**

### CODE REVIEW

File: app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 import androidx.annotation.StringRes
 import com.vero.tasky.R
 import com.vero.tasky.agenda.domain.model.AgendaItem
+import com.vero.tasky.agenda.domain.model.Attendee
 import com.vero.tasky.agenda.presentation.model.ReminderRange
+import kotlinx.parcelize.Parcelize
 import java.time.LocalDateTime
 import java.util.*
-
+@Parcelize
```
Comment (MKiperszmid) [inline app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt:12]: Why do you need the state to be parcelable?

File: app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsViewModel.kt [hunk 2]
```diff
@@ -168,11 +201,14 @@ class EventDetailsViewModel @Inject constructor(
...
                         val agendaItem = state.agendaItem
+                        val attendees = agendaItem.attendees + newAttendee
                         updateState(
                             state.copy(
                                 agendaItem = agendaItem.copy(
-                                    attendees = agendaItem.attendees + newAttendee
+                                    attendees = attendees
                                 ),
+                                isGoingAttendees = attendees.filter { it.isGoing },
+                                isNotGoingAttendees = attendees.filter { !it.isGoing },
```
Comment (MKiperszmid) [inline app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsViewModel.kt:211]: You are running this filter in several places (here and on init)
You could have a "filterAttendees" or similar and run it there

File: app/src/main/java/com/vero/tasky/core/presentation/navigation/RootNavigation.kt [hunk 3]
```diff
@@ -66,26 +72,75 @@ fun RootNavigation(
...
+            )
+        ) {
+            val text = it.arguments?.getString(EDIT_DESCRIPTION_VALUE) ?: ""
+            EditTextScreen(
+                label = R.string.edit_description,
+                textValue = text,
+                navBack = { navController.popBackStack() },
+                save = { resultStr ->
+                    navController.previousBackStackEntry
+                        ?.savedStateHandle
```
Comment (MKiperszmid) [inline app/src/main/java/com/vero/tasky/core/presentation/navigation/RootNavigation.kt:129]: You are creating 2 routes. One for title and one for description

You could just have 1 route, with 2 optional query parameters 

So if the query parameter "title" is set, then you edit the title. Else, it's the description

**Review by MKiperszmid — APPROVED** [review]

File: app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt (hunk 1 above)
Comment (AndreVero) [reply app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt:12]: To save state in the SavedStateHandle

Comment (MKiperszmid) [reply app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/EventDetailsState.kt:12]: Gotcha

**Pull request merged**